scripts/run_on_popular_crates/src/main.rs

```diff
@@ -0,0 +1,127 @@
+// use std::io;
+// use std::fs::File;
+// use core::error::Error;
+use serde::Deserialize;
+use std::collections::BTreeMap;
+use std::path::Path;
+
+fn download_crates_data() -> String {
+    println!("Downloading db-dump.tar.gz (it can take a while)");
+    let resp = reqwest::blocking::get("https://static.crates.io/db-dump.tar.gz").unwrap();
```

Comment:
Do we really need the full dump? Can't we just use an API client to fetch the most-downloaded crates directly, like this: https://docs.rs/crates_io_api/0.8.1/crates_io_api/struct.CratesQuery.html#method.set_sort

Reply:
Oh, so this is where the API is. I couldn't find it on the crates.io.